packages/experimental/src/components/Card/_storybook-styles.scss

```diff
@@ -1,8 +1,45 @@
+/* stylelint-disable */
+
 @import './index';
+@import '../../global/styles/project-settings';
+@import '../../global/styles/carbon-settings';
 
 // make the root full width to get a better idea of how the cards
 // look in a real grid situation
 
 #root {
   width: 100%;
 }
+
+// solution for image ratios- generate empty boxes to represent media
+
+.media-box {
+  width: 100%;
+  background: $ui-03;
+}
+
+.media-ratio-11 .media-box {
+  padding-bottom: 100%;
+}
+
+.media-ratio-43 .media-box {
+  padding-bottom: 75%;
+}
+
+.media-ratio-32 .media-box {
+  padding-bottom: 66.66%;
+}
+
+.media-ratio-169 .media-box {
+  padding-bottom: 56.25%;
```

Comment:
I wonder if we could use [`aspect-ratio`](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) here?

Reply:
Or even https://react.carbondesignsystem.com/?path=/story/aspectratio--playground

Reply:
oh wow i can't believe they already had a component for this built. imagine my surprise. i'll go ahead and see if I can use either of these to improve this. thanks!